api/src/wfl/api/workloads.clj

```diff
@@ -11,27 +11,34 @@
 ;; creating and dispatching workloads to cromwell
 (defmulti create-workload!
   "(transaction workload-request) -> workload"
-  (fn [_ body] (:pipeline body)))
+  (fn [transaction request] (:pipeline request)))
```

Comment:
`_transaction` will suppress a linter warning, and so on.
This parameter is named `tx` or `_tx` everywhere else.

Reply:
tx is a terrible name
I'll add an underscore to make the linter happy.

Reply:
OK, but now we have at least 2 different names for the same thing.

Reply:
I think we should write `transaction` in interface functions. I don't really care what it's called in implementations or private functions.

Reply:
OK, Fred.